notebooks/00_quick_start/rlrmc_movielens.ipynb

```diff
@@ -0,0 +1,267 @@
+{
```

Comment:
<p>can you add some metrics here to get a sense of how this performs? see examples in other notebooks (like <a href="https://github.com/Microsoft/Recommenders/blob/master/notebooks/02_model/baseline_deep_dive.ipynb" target="_blank">https://github.com/Microsoft/Recommenders/blob/master/notebooks/02_model/baseline_deep_dive.ipynb</a>)</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/729/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
done.